PKHeX.WinForms/MainWindow/Main.cs

```diff
@@ -103,7 +106,7 @@ private set
         private static string TemplatePath => Path.Combine(WorkingDirectory, "template");
         private static string PluginPath => Path.Combine(WorkingDirectory, "plugins");
         private const string ThreadPath = "https://projectpokemon.org/pkhex/";
-        private const string VersionPath = "https://raw.githubusercontent.com/kwsch/PKHeX/master/PKHeX.WinForms/Resources/text/version.txt";
+        private const string VersionPath = "https://raw.githubusercontent.com/kwsch/PKHeX/master/version.txt";
```

Comment:
Since the old file can't go away, maybe I should just go ahead and move the new version to a subdomain of projectpokemon.org or something.